docs/src/modules/components/AppDrawerNavItem.js

```diff
@@ -34,8 +34,7 @@ const styles = theme => ({
     },
   },
   active: {
-    color:
-      theme.palette.type === 'light' ? theme.palette.primary.dark : theme.palette.primary.light,
+    color: theme.palette.primary.main,
```

Comment:
Using the light color for the dark theme was good, though. Let's half-un-revert. :smile:

Reply:
My concern was more about having the documentation hiding the problem when the core components suffers from it.

Reply:
I can see the problem, but the color (and thus contrast) matches the spec and is "almost" AA.